contracts/Makefile

```
@@ -1,13 +1,14 @@
 # Stellar Portfolio Rebalancer Contract Makefile
 
-.PHONY: build test bench clean deploy-testnet deploy-mainnet setup-testnet
+.PHONY: build test bench clean deploy-testnet deploy-mainnet setup-testnet hash
 
```

checkmake (0.3.2)

[warning] Required target "all" must be declared PHONY.

(minphony)